editor-components/feature/feature9/feature9.tsx

```diff
@@ -1,192 +1,280 @@
+        nextElement.style.opacity = '0';
+      }
+    } else {
+      counterElement.classList.add(this.decorateCSS("invisible"));
+      currentElement.style.opacity = '0';
+      
+      if (nextElement) {
+        nextElement.style.opacity = '0';
+      }
+    }
+
+    if(currentElement.id=="card-1") {
+      counterElement.classList.remove(this.decorateCSS("invisible"));
+    }
+  }
+
+  updateCounter(id: number) {
+    const counterElement = document.getElementById("counter");
+    if (counterElement) {
+      counterElement.innerText = id.toString();
+      counterElement.classList.remove(this.decorateCSS("invisible"));
+    }
+  }
+
+  renderCard(featureItem: FeatureItem, index: number) {
+    return (
+      <div 
+        key={index} 
+        id={`card-${index}`} 
+        className={this.decorateCSS("card-item")}>
+        <img
+          alt="description image" 
+          className={this.decorateCSS("image")}
+          src={featureItem.image}
+        />
+        <h4 className={this.decorateCSS("sub-title")}>{featureItem.title}</h4>
+        <h5 className={this.decorateCSS("sub-title-description")}>
+          {featureItem.description}
+        </h5>
+        <br /><br/><br/><br /><br/><br/>
+      </div>
+    );
+  }
+
   render() {
+    setTimeout(() => this.setupObservers(), 0);
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
           <div className={this.decorateCSS("content")}>
-            {this.castToObject<Horizontals[]>("horizontal").map(
-              (horizontals: any, index: number) => (
-                <div className={this.decorateCSS("card-item-count")} style={{
-                  width: 90 / this.getPropValue("itemCount") + "%",
-                }}>
-                <div className={this.decorateCSS("horizontal")} key={index}>
-                  <img
-                    alt=""
-                    className={this.decorateCSS("image")}
-                    src={horizontals.image}
-                  ></img>
-                  <div className={this.decorateCSS("block")}>
-                    <h3 className={this.decorateCSS("title")}>
-                      {horizontals.title}
-                    </h3>
-                    <p className={this.decorateCSS("title-description")}>
-                      {horizontals.description}
-                    </p>
-                    <ComposerLink path={horizontals.link}>
-                      <span className={this.decorateCSS("button")}>
-                        {horizontals.buttonText}
-                      </span>
-                    </ComposerLink>
-                  </div>
-                </div>
-                </div>
-              )
-            )}
+            <div className={this.decorateCSS("main-title-content")}>
+              <h1 className={this.decorateCSS("main-title-content-text")}>
+                {this.getPropValue("main-title")}
+              </h1>
+            </div>
```

Comment:
don't render this if prop is empty
[read here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)